src/py3.10/Pipfile

```
@@ -0,0 +1,11 @@
+[[source]]
+url = "https://pypi.org/simple"
+verify_ssl = true
+name = "pypi"
```

This is probably better as a separate PR, but we should alphabetize these per our usual standards:
```suggestion
name = "pypi"
url = "https://pypi.org/simple"
verify_ssl = true
```

---

If you're amenable I'll make a new branch + PR based on this branch fixing all four Pipfiles.

---

That works for me, thanks!

---

Just mention the new PR here when it's created and then I'll resolve this conversation.

---

This PR's work is going to be superseded by implementing https://github.com/cisagov/skeleton-aws-lambda-python/issues/15. The suggestion here will be integrated into the PR that resolved the linked issue.